docs/Homebrew-and-Python.md

```diff
@@ -11,13 +11,19 @@ Homebrew provided a `python@2` formula until the end of 2019, at which point it
 **Important:** If you choose to use a Python which isn't either of these two (system Python or brewed Python), the Homebrew team cannot support any breakage that may occur.
 
 ## Python 3.x
-Homebrew provides a formula for Python 3.x (`python`).
+Homebrew provides a formula for Python 3.x (`python@3.x`).
 
 The executables are organised as follows:
 
 * `python3` points to Homebrew's Python 3.x (if installed)
 * `pip3` points to Homebrew's Python 3.x's pip (if installed)
 
+Unversioned symlinks for `python`, `python-config`, `pip` etc. are installed here:
+
+```sh
+$(brew --prefix)/opt/python/libexec/bin
```

Comment:
There are many options for this path:

- `$(brew --prefix)/opt/python/libexec/bin`
- `$(brew --prefix)/opt/python3/libexec/bin`
- `$(brew --prefix)/opt/python@3/libexec/bin`
- `$(brew --prefix)/opt/python@3.x/libexec/bin`

I chose `python` as it shouldn't depend on any version-specific structures (which means you won't have to change your `PATH` for new versions of python, although `python3` or `python@3` would also accomplish this for Python 3 versions), but let me know if I should use a different one